### DISCUSSION TITLE
how i can fetch historical-weather-api in reactJs

### DISCUSSION THREAD

**Discussion by Kietnhn** [open-meteo/open-meteo#253] [Q&A]
i try to fetch('https://archive-api.open-meteo.com/v1/archive?latitude=52.52&longitude=13.41&start_date=2023-02-05&end_date=2023-03-07&hourly=temperature_2m') in my project but response is { size: 0 }.How to fetch with fully data like website

**Comment by patrick-zippenfenig** [comment]
The request is fine and the API fully supports HTTP requests from different origins (CORS). Can you share your code?
